### ISSUE TITLE
Compliment

### ISSUE THREAD

**Issue by DeiseFreire** [StefanHritcu/stefanhritcu#1]
The design of your Github profile README is super professional. The combination of colors and fonts is very nice.
The use of emojis makes your profile README more fun and humanized. Thanks for following me, neighbor! ❤️

**Comment by StefanHritcu** [comment]
Thank you so much, that's very kind of you! Where in Cologne are you located?

Here’s my portfolio—I finished it just two days ago. I’d love to hear your thoughts! 😊

https://hritcu-personal-portfolio.netlify.app/

P.S. What’s your focus area in programming?

**Comment by DeiseFreire** [comment]
Hello neighbor, @StefanHritcu! I live in Brasilia, Brazil and I have a degree in Internet Systems. I am currently studying English Language. Your portfolio is a work of art, it is stunningly beautiful. The combination of colors and fonts is very nice. Let's be neighbors on YouTube too, I just subscribed to your channel and I like everything.

**Comment by StefanHritcu** [comment]
Thank you so much, you're very kind. I'll do the same. Do you have LinkedIn?

We could promote each other, it helps.

I also noticed that you're studying German. Do you want to come to Germany?

**Comment by DeiseFreire** [comment]
Yes neighbor @StefanHritcu, I would like to live in Germany ❤️ . And I have a profile on LinkedIn too. It will be an honor to have you as my neighbor on LinkedIn. @StefanHritcu many follow you, but only I accompany you (GitHub, Youtube, LinkedIn)

[![LinkedIn](https://img.shields.io/badge/linkedin-%230077B5.svg?style=for-the-badge&logo=linkedin&logoColor=white)](https://www.linkedin.com/in/deise-freire-3a7798212/)

**Comment by StefanHritcu** [comment]
Hey. I'm currently in Italy working with acquaintances and friends, doing cards, and I'm just involved in furniture projects... How's it going on your end?

Based on the data, how long will it take for you to get to Germany, and why Germany?

Good morning, I think. I'm going to sleep.

**Comment by DeiseFreire** [comment]
Hello @StefanHritcu, I am studying German. If you have any grammar in PDF or digital format that you could share with me, I would be very grateful. You can send it to me by message via LinkedIn or here. @StefanHritcu, my natural habitat is the gym. Any country in the world that has a gym with weights, plates, and training equipment is wonderful for me. Since I study German, I dream of visiting Germany.
